lib_parts.ks

```diff
@@ -0,0 +1,21 @@
+FUNCTION ExtendAntennas {
```

Comment:
might want to prefix with `parts` just in case another program redefines this function. (This happened to me once, which is why I began prefixing library functions with `ui`, `util`, etc.)

Reply:
I'll do that, for the sake of keeping a standard across the entire project.

Reply:
On that topic: if you feel like refactoring and changing the naming convention, don't hestitate to do that. (e.g. if you prefer CamelCase to snakeCase, or you don't like the `_` in names of things.) At this point, you're the primary maintainer and I'm just an advisor. ;-)

Reply:
I typed a lot of things in caps lock (to follow the kos documentation) but them I found that ugly. 
I think in weekend I will have time to do a pass in all code to check this, the identations, variable names, commens, such things. Or at least in the files I use most.